states/oklahoma/main.py

```diff
@@ -0,0 +1,17 @@
+import parse_pdf
+import parse_html_emails
+import json
+
+if __name__ == "__main__":
+    # Get all data except emails
+    counties = parse_pdf.main()
+    # Get all emails.
+    emails = parse_html_emails.main()
+
+    assert len(counties) == len(emails)
+
+    for i in range(0, len(counties)):
+        counties[i]['emails'].append(emails[i])
```

Comment:
This is really bad.  In general, I wouldn't want to rely on the two lists being aligned.  It's a subtle bug to catch. 
 Instead, let's join on county name (basically, create a dict with one and select in with the other).

Reply:
Yeah...you're right, I'm sorry.